src/linter/root.go

```diff
@@ -501,6 +502,7 @@ func (d *rootWalker) parseFuncParams(params []ir.Node, docblockParams phpdoctype
 		return d.parseFuncArgsForCallback(params, sc, closureSolver)
 	}
 
+	isVariadic := false
```

Comment:
move above where `paramHints` is declared

Reply:
Ok